### DISCUSSION TITLE
[Hackathon Use Case] - Product Control Template - Testing Environment

### DISCUSSION THREAD

**Discussion by ffionwiggins** [finos/finos-pmcs#248] [Ideas]
# Product Control Template - Testing Environment
_From @ffionwiggins (Goldman Sachs)_

## Business Problem

Over the past 18months the FINOS FO Community have been collaborating on the build of a Unified Product Control Template. This template contains the necessary attributes for submitting price transparency data to certain industry vendors which historically has multiple formats and content based on vendor. 

The model/template is now available in Legend and the working group have moved to testing phase, however this is proving difficult without a dummy implementation. 

The request is for a testing environment to be set up such that companies could test mapping their internal data to the new template and submitting [this should be dummy data initially].

## Expected features
TBD

## Useful apps, systems, projects, technologies, libraries and datasets
- [FINOS Financial Objects](https://fo.finos.org)
- [Product Control Common Template](https://www.finos.org/blog/project-updates-financial-objects)
- [Legend](https://legend.finos.org)

**Comment by TheJuanAndOnly99** [comment]
Closing discussion in preparation [for this years hackathon](https://www.finos.org/hosted-events/2024-10-25-citi-india-hackathon-supported-by-finos). Feel free to reopen and update if its still relevant.
